README.md

```diff
@@ -1,602 +1,387 @@
+  | [model][]        |                                    |
+  | [controller][]\* | [`ActionController::TestCase`][]   |
+  | [mailer][]       | `ActionMailer::TestCase`           |
+  | [job][]          |                                    |
+  | [view][]         | `ActionView::TestCase`             |
+  | [routing][]      |                                    |
+  | [helper][]       | `ActionView::TestCase`             |
+
+- **Integration**
+
+  | Spec type       | Corresponding Rails test class        |
+  | --------------- | ------------------------------------- |
+  | [request][]     | [`ActionDispatch::IntegrationTest`][] |
+  | [feature][]\*   |                                       |
+  | [system][]      | [`ActionDispatch::SystemTestCase`][]  |
+
+_\* discouraged in favor new alternatives_
+
+Follow the links above to see examples of each spec type,
+or for official Rails API documentation on the given `TestCase` class.
+
+> **Note: This is not a checklist.**
+> 
+> Ask a hundred developers how to test an application,
+> and you’ll get a hundred different answers.
+> Some may insist on exhaustively testing every single method on every model;
+> others may contend that since integration tests capture model behavior anyway,
+> much unit testing is redundant and burdensome.
+>
+> RSpec Rails provides thoughtfully selected features
+> to encourage good testing practices, but there’s no “right” way to do it.
+> Ultimately, it’s up to you to decide how your test suite will be composed.
+> 
+> (You may even define custom types, or write specs with no type at all.)
+
+[request]: https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec
+[feature]: https://www.relishapp.com/rspec/rspec-rails/docs/feature-specs/feature-spec
+[system]: https://relishapp.com/rspec/rspec-rails/docs/system-specs/system-spec
+[model]: https://www.relishapp.com/rspec/rspec-rails/docs/model-specs
+[controller]: https://www.relishapp.com/rspec/rspec-rails/docs/controller-specs
+[mailer]: https://relishapp.com/rspec/rspec-rails/docs/mailer-specs
+[job]: https://relishapp.com/rspec/rspec-rails/docs/job-specs/job-spec
+[view]: https://www.relishapp.com/rspec/rspec-rails/docs/view-specs/view-spec
+[routing]: https://www.relishapp.com/rspec/rspec-rails/docs/routing-specs
+[helper]: https://www.relishapp.com/rspec/rspec-rails/docs/helper-specs/helper-spec
+[`ActionDispatch::IntegrationTest`]: https://edgeapi.rubyonrails.org/classes/ActionDispatch/IntegrationTest.html
+[`ActionDispatch::SystemTestCase`]: https://edgeapi.rubyonrails.org/classes/ActionDispatch/SystemTestCase.html
+[`ActionController::TestCase`]: https://edgeapi.rubyonrails.org/classes/ActionController/TestCase.html
+
+### I’m writing a spec file from scratch. How do I assign it a type?
+
+Assuming you haven’t [modified the default `rails_helper.rb` configuration][],
+simply place the spec in the appropriate folder
+(_e.g.,_ `spec/models/` for model specs)
+and RSpec will set its type automatically.
```

Comment:
Don't mention this, we don't want to encourage it.

Reply:
Got it.

Reply:
I'd still like this changed to describe assigning a type by hand, and not mentioning setting type automatically.

Reply:
This still needs inverting. The default should be writing types, that it can happen automatically is likely going to be removed in a later RSpec version, so it should not be mentioned.

Reply:
Sorry @JonRowe, just noticed this now. Does the current version work for you?